frame/benchmarking/src/tests.rs

```diff
@@ -36,7 +36,7 @@ mod pallet_test {
 
 	#[pallet::pallet]
 	#[pallet::generate_store(pub(super) trait Store)]
-	pub struct Pallet<T>(_);
+	pub struct Pallet<T>(PhantomData<T>);
```

Comment:
```suggestion
	pub struct Pallet<T>(_);
```

Reply:
Why not use `struct Pallet<T>(PhantomData<T>);`?

`struct Pallet<T>(_);` is not standard rust syntax

Reply:
Yeah I know. This syntax was added as some shortcut.

Reply:
i would not be against removing these shortcuts if you find it confusing. End of the day, no one writes this line by hand, they just copy some template.